caffe_app/views/import_prototxt.py

```diff
@@ -2,25 +2,30 @@
 from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse
 import yaml
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
There is no need to declare the BASE_DIR again. You can just import it  

`from django.conf import settings`
And then use it simply as `settings.BASE_DIR`. 

For more information about this, please see https://docs.djangoproject.com/en/1.11/topics/settings/#using-settings-in-python-code